src/lib/testUtils.js

```diff
@@ -12,11 +12,10 @@ module.exports = {
 
   createTestUserOpts: (number = 0) => {
     const randomNum = _.random(10000) + number;
-    const name = `name${randomNum}`;
+    const id = `name${randomNum}`;
```

Comment:
Is there any reason we're removing the `name` field?. Think we should keep it in there

Reply:
This has been fixed in a later PR, we will ignore these changes when merging branches.